client/src/app/constants/appPaths.js

```diff
@@ -1,12 +0,0 @@
-export const appPaths = [
-  {
-    label: 'Overview',
-    icon: 'visibility',
-    path: '#overview',
-  },
-  {
-    label: 'Support',
-    icon: 'info',
-    path: '#support',
-  },
-]
```

Comment:
Can't we use the appPaths in the new NavMenu?

Reply:
I think that writing JSX is easier and more familiar than writing JSON with an undocumented structure.